config/formats.ts

```diff
@@ -379,7 +379,7 @@ export const Formats: import('../sim/dex-formats').FormatList = [
 		desc: `A Gen 1 metagame where Fakemon are added in a similar fashion to Smogon's official Create-A-Pokemon Project.`,
 		mod: 'gen1rbycap',
 		ruleset: ['Standard'],
-		banlist: ['Uber'],
+		banlist: ['Uber', 'Camouflage'],
```

Comment:
why?

Reply:
Camouflage was added as a signature move for Pineguin, but instead of its effects in vanilla later gens, it's a Grass-type Dig clone. Dig and Fly are banned in Gen 1 because of the Desync glitch, so I banned Camouflage to be in line with that, but I just put it in the banlist instead of making a rulesets file to add it to Standard.

Reply:
ok thats fine, just didnt want it to be banned bc of a crash or something